perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -178,7 +177,7 @@ public Member createSpecificMember(PerunSession sess, Vo vo, Candidate candidate
 			}
 		}
 
-		return getMembersManagerBl().createSpecificMember(sess, vo, candidate, specificUserOwners, specificUserType, groups);
+		return getMembersManagerBl().createServiceMember(sess, vo, candidate, specificUserOwners, groups);
```

Comment:
You should add exception to the RPC when someone is trying to create specific member which is not service member.

Reply:
There is already an exception thrown on line 156.

Reply:
My bad, didn't see that.